pkg/api/v1/pod/util.go

```diff
@@ -25,6 +25,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+// Pod begins existence as unready. If pod is unready and timestamp of last pod readiness change is
+// less than maxDelayOfInitialReadinessStatus after pod start we assume it has never been ready.
+const maxDelayOfInitialReadinessStatus = 10 * time.Second
```

Comment:
this is fairly HPA specific, so should probably go in the HPA utils, especially since it's a heuristic and not a concrete check

Reply:
same with the func

Reply:
Done.